src/app/plugins/pubsub.plugin.ts

```diff
@@ -17,15 +19,16 @@ export class PubSubPlugin extends Plugin {
   private _SUBS: string[] = [];
   private _EMBED_LIST = new MessageEmbed();
 
-  private _SUB_UPD_THRESH: number = 1000 * 60 * 60 * 24; // in ms, one day.
+  private _PUB_SUB_EMOJI: GuildEmoji | string = '';
```

Comment:
but it can be a string, although I don't know why I didn't make it 🥪 to begin with.

line45:      `message?.guild?.emojis.cache.filter((e) => e.name === 'pubsub').first() || '🥪';`

Reply:
Make it a Maybe<GuildEmoji> then just `message.send(emoji || '🥪');`